### DISCUSSION TITLE
Is there a way to force all users to receive email notifications of media available/approved/declined?

### DISCUSSION THREAD

**Discussion by 2fst4u** [sct/overseerr#2075] [Q&A]
I've only just realised I don't think emails are being sent to my users when their media is available. I get them myself because I've enabled the notifications for me, but they aren't global, it's per user.

Is there a way to force these notifications to send to users when they request things without having to tell them to edit their own settings?

**Comment by TheCatLady** [comment]
Email notifications are enabled for all users by default, so if a user doesn't have them enabled, it means that they (or an admin) deliberately modified their settings.
